roles/technical_principal.md

```diff
@@ -0,0 +1,154 @@
+
+### To coach and support teams running workshops and roadmapping sessions
+
+We expect our Technical Principals to coach teams to be able to run workshops and roadmapping sessions, to provide a supporting role in important sessions, and foster collaboration with the wider team to identify a technology roadmap to solve business problems.
+
+### To run technical discovery projects
+
+We expect our Technical Principals to run technical discoveries. Technical Principals are responsible for planning, workshopping, running technical feasibility studies, and ultimately recommending strategies to customers.
+
+### To be partner with customer tech leadership (CTO/CIO) to help shape their technology strategies
+
+We expect our Technical Pricipals to be able to engage the highest levels of technical leadership within our customers organisations. Through building strong relationships Technical Principals will be seen as trusted advisors.
+
+## Commercially Aware Engineer Expectations
+
+### To quickly become a trusted partner to the customer team, building a friendly relationship
+
+We expect our commercially aware engineers to quickly become a trusted partner to the customer team - both our day-to-day contacts, and more senior stakeholders. A commercially aware engineer should be able to put a customer at ease by engaging in small talk, and by building strong and friendly working relationships. A commercially aware engineer is expected to credibly and knowledgeably represent Made Tech when conversing with senior stakeholders.
+
+### To proactively identify opportunities where we can widen the impact of our work with the customer organisation
+
+We expect our commercially aware engineers to proactively seek opportunities where we can extend our remit with the customer organisation. A commercially aware engineer is expected to understand the objectives of our stakeholders, and to sell Made Tech’s wider capabilities.
+
+### To make sensible, well reasoned commercial decisions
+
+We expect our commercially aware engineers to proactively make good commercial decisions. A commercially aware engineer is expected to identify and course correct potential delivery issues before they become impactful. A commercially aware engineer is expected to proactively escalate larger delivery issues to the Delivery Manager or Delivery Director.
+
+## Senior Engineer Expectations
+
+### To make sensible and well justified technical architecture decisions, involving the team in the decision making process where appropriate
+
+We expect our Senior Engineers to be able to design and implement appropriate technical architectures to solve problems. We expect them to appropriately involve the team in the decision making process to help them develop their technical architecture skills.
+
+### To have attained the Made Tech Core Skills
```

Comment:
Perhaps controversially I'm unsure this is necessary at the TP level

Reply:
I see some tension here. "Leading by Example" - can cause tensions if people are questioning the ability of the people who are meant to be mentoring them. 

In an ideal world, technical principals would be highly respected for their ability to straddle two worlds seemingly effortlessly.

Reply:
I've removed core and senior role expectations.